Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -345,6 +341,12 @@ def service_binding_create_command(client: PolicyComputeEngine, args: dict[str,
 
     validate_required_parameters(workloads=workloads, virtual_service=virtual_service)
     workloads = argToList(workloads)
+    virtual_service = convert_draft_href_to_active(args.get("virtual_service"))
+    try:
+        _ = client.virtual_services.get_by_reference(virtual_service)
```

Comment:
There's no need for the left side of this assignment

Reply:
ok